mleap-grpc-server/src/test/scala/ml/combust/mleap/grpc/server/GrpcSpec.scala

```diff
@@ -6,12 +6,14 @@ import akka.testkit.TestKit
 import io.grpc.{ManagedChannel, Server}
 import ml.combust.mleap.executor.service.TransformService
 import ml.combust.mleap.executor.testkit.TransformServiceSpec
-import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Ignore}
 import org.scalatest.concurrent.ScalaFutures
 
 import scala.concurrent.duration._
 import ml.combust.mleap.grpc.server.TestUtil._
 
+// Ignored temporarily because of random failures on Travis
+@Ignore
```

Comment:
I believe the flaky one is the "throttling" in TransformServiceSpec, could you please try ignoring just that for now? thank you!

Reply:
Changed to ignore "throttling" only